portlet-tck_3.0/driver/src/main/java/javax/portlet/tck/driver/TCKSimpleTestDriver.java

```diff
@@ -228,15 +233,21 @@ public static void setUpBeforeClass() throws Exception {
       System.out.println("   PasswordId   =" + passwordId);
       System.out.println("   Browser      =" + browser);
       System.out.println("   Driver       =" + wd);
+      System.out.println("   binary       =" + binary + " ... used for ChromeDriver & FirefoxDriver only, for now");
 
       if (browser.equalsIgnoreCase("firefox")) {
-         driver = new FirefoxDriver();
+         driver = new FirefoxDriver(new FirefoxBinary(new File(binary)), new FirefoxProfile());
       } else if (browser.equalsIgnoreCase("internetExplorer")) {
          System.setProperty("webdriver.ie.driver", wd);
          driver = new InternetExplorerDriver();
       } else if (browser.equalsIgnoreCase("chrome")) {
          System.setProperty("webdriver.chrome.driver", wd);
-         driver = new ChromeDriver();
+         ChromeOptions options = new ChromeOptions();
+         options.setBinary(binary);
+         options.addArguments("--headless");
+         options.addArguments("--disable-infobars");
+         driver = new ChromeDriver(options);
+         System.out.println("!! option       = headless");
```

Comment:
@vsingleton Is there a way to disable *headless* chrome like `-Dintegration.browser.headless=false` in the Liferay Faces Test framework? https://github.com/ngriffin7a/liferay-faces-test-selenium/blob/663c1bd78b53f3746333c5b191d9eec6faabef0c/pom.xml#L105
If not then need to add that.